ui/app/models/pki/config.js

```diff
@@ -12,19 +12,20 @@ export default class PkiConfigModel extends Model {
     return this.secretMountPath.currentPath;
   }
 
-  @lazyCapabilities(apiPath`${'backend'}/config/ca`, 'backend') configCaPath;
-  @lazyCapabilities(apiPath`${'backend'}/root/generate/${'type'}`, 'backend', 'type') generateRootPath;
-  @lazyCapabilities(apiPath`${'backend'}/intermediate/generate/${'type'}`, 'backend', 'type') generateCsrPath;
-  @lazyCapabilities(apiPath`${'backend'}/intermediate/cross-sign`, 'backend') crossSignPath;
+  @lazyCapabilities(apiPath`${'backend'}/issuers/import/bundle`, 'backend') importBundlePath;
+  @lazyCapabilities(apiPath`${'backend'}/issuers/generate/root/${'type'}`, 'backend', 'type')
+  generateIssuerRootPath;
+  @lazyCapabilities(apiPath`${'backend'}/issuers/generate/intermediate/${'type'}`, 'backend', 'type')
```

Comment:
Just a note for later - I think both of these generate capability paths will have to be without `type` since that's selected by the user in the form and so it will always be undefined since `type` won't exist on the model until it's selected?